provisioning/miniprov/hare_mp/templates/hare.config.conf.tmpl.1-node

```diff
@@ -21,6 +21,24 @@
       ]
     }
   },
+  "cortx": {
+    "software": {
+      "motr": {
+        "service": {
+          "client_instances": "2"
+        }
+      }
+    }
+  },
+  "cortx": {
+    "software": {
+      "s3": {
+        "service": {
+          "instances": "TMPL_S3SERVER_INSTANCES_COUNT"
+        }
+      }
+    }
+  },
```

Comment:
```suggestion
  "cortx": {
    "software": {
        "motr": {
          "service": {
            "client_instances": "2"
          }
        },
        "s3": {
          "service": {
            "instances": "TMPL_S3SERVER_INSTANCES_COUNT"
          }
        }
    }
  },
```
@supriyachavan4398 
motr : value  and s3: value, should come under same cortx>software... path ?

Reply:
Yes, we can add these values under the same path cortx>software... path.